src/components/uicomposition/Quote.astro

```diff
@@ -23,13 +23,16 @@ const variant = testimonial.variant ? testimonial.variant : "glass";
   {
     variant == "flat" && (
       <div class="flex flex-col items-center justify-center text-white md:w-4/12">
-        <SanityImg
-          src={testimonial.person?.image?.asset?._id || ""}
-          alt={testimonial.person?.image?.asset?.altTexts?.[locale] || ""}
-          class="mb-8 aspect-square h-52 w-52 rounded-full transition-transform duration-300 ease-in-out motion-safe:group-hover/quote:scale-110 sm:h-40 sm:w-40 lg:h-52 lg:w-52"
-          loading="lazy"
-          sizes="(min-width:768px) 50vw, 100vw"
-        />
+        {testimonial.person?.image?.asset?._id && (
+          <SanityImg
+            src={testimonial.person?.image?.asset?._id}
```

Comment:
the component looks a bit weird without an image. Mauybe it's better to upload a placeholder image and reference that? I think for both the quote and casepreview compnent. For the cardgrid it's not that important

Reply:
We should make a new Jira ticket for this and ask Manon for a default picture design.

Reply:
sure, in that case this is fine ✅